homework-1.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title></title>
+
+    <!-- Bootstrap -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
+  </head>
+  <body>
+  <h2>Add Item</h2>
+  <h3> Item Name: </h3> <input id="name" placeholder="Item Name" width="400px"><br>
+  <h3> Price: </h3><input id="price" placeholder="Price don't use a dollar sign"><br>
```

Comment:
`<input>` needs a type. In this case you can give it `number` which won't allow the $.